vlf-tune.el

```diff
@@ -35,6 +35,11 @@
   :group 'vlf :type 'integer)
 (put 'vlf-batch-size 'permanent-local t)
 
+(defcustom vlf-batch-size-remote 1024
+  "Defines size (in bytes) of a batch of file data when accessed remotely."
+  :group 'vlf :type 'integer)
+(put 'vlf-batch-size-remote 'permanent-local t)
```

Comment:
permanent-local is not needed in this case.  This customization is to
be used only once per VLF invocation, setting _initial_ value
vlf-batch-size, which has life on its own later.